.circleci/config.yml

```diff
@@ -80,3 +81,5 @@ workflows:
           requires:
             - test
           <<: *tag_filter
+
+# VS Code Extension Version: 1.5.1
```

Comment:
@Remi-Gau just a note that this comment seems to get inserted by some extension of yours, which is undesirable I guess :-)

Reply:
oops indeed